examples/deck/example.js

```diff
@@ -92,8 +92,14 @@ export default class Example extends Component<
 
   _onLayerClick = info => {
     console.log('onLayerClick', info); // eslint-disable-line
+    const { mode } = this.state;
 
-    if (this.state.mode !== 'view') {
+    if (mode === 'drawPoint') {
```

Comment:
For now, I don't think we need this check. A consuming application can do this kind of "bulk draw points" mode by leaving `selectedFeatureIndexes=[]` after the `onEdit` callback.